extensions/wikia/ThemeDesigner/ThemeDesignerController.class.php

```diff
@@ -262,25 +261,26 @@ public function executeBackgroundImageUpload() {
 
 		$upload = new UploadBackgroundFromFile();
 
-		$status = $this->uploadImage($upload);
+		$status = $this->uploadImage( $upload );
 
-		if($status['status'] === 'uploadattempted' && $status['isGood']) {
-			$file = $upload->getLocalFile(); /* @var $file LocalFile */
+		if ( $status['status'] === 'uploadattempted' && $status['isGood'] ) {
+			$file = $upload->getLocalFile();
+			/* @var $file LocalFile */
 			$this->backgroundImageUrl = wfReplaceImageServer( $file->getUrl() );
 			$this->backgroundImageName = $file->getName();
 			$this->backgroundImageHeight = $file->getHeight();
 			$this->backgroundImageWidth = $file->getWidth();
 
 			//get cropped URL
-			$is = new ImageServing( null, 120, array( "w"=>"120", "h"=>"100" ) );
+			$is = new ImageServing( null, 120, [ "w" => "120", "h" => "100" ] );
 			$this->backgroundImageThumb = wfReplaceImageServer( $file->getThumbUrl( $is->getCut( $file->width, $file->height, "origin" ) . "-" . $file->getName() ) );
 
 			// if background image url is not set then it means there was some problem
 			if ( $this->backgroundImageUrl == null ) {
-				$this->errors = array( wfMsg( 'themedesigner-unknown-error' ) );
+				$this->errors = [ wfMsg( 'themedesigner-unknown-error' ) ];
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)